src/components/ui/Button.tsx

```diff
@@ -30,7 +33,8 @@ export function Button(props: ButtonProps): JSX.Element {
 export function createButtonProps(
   type: 'submit' | 'reset' | 'button' | undefined,
   name: string,
-  color: string
+  color: string,
+  disabled?: boolean
 ): Button {
-  return { type, name, color };
+  return { type, name, color, disabled };
```

Comment:
`style={{ backgroundColor: button.color }}` の部分を消していると思うので、これも消しましょう〜！3

```suggestion
  return { type, name, disabled };
```